lib/experimentUtils.ts

```diff
@@ -65,4 +65,50 @@ export function getProposedEndDate (startDate : string | null, proposedDuration
   const formattedDate = jsDate.toISOString().slice(0, 10)
 
   return formattedDate
-}
\ No newline at end of file
+}
+
+/**
+ * Do recursive locale substitution on the values, if applicable.
+ *
+ * If there are no localizations provided, the value will be returned as-is.
+ *
+ * If the value is an object containing an $l10n key, its substitution will be
+ * returned.
+ *
+ * Otherwise, the value will be recursively substituted.
+ * 
+ * Right now, we are manually passing the first locale out of the localization object.
+ * Eventually we'll want to select a locale on the dashboard (probably with a dropdown.)
+ *
+ * @param values - The values to perform substitutions upon; message contents
+ * @param localizations - The localization object from the recipe, contains
+ *        substitutions for a specific locale. May be null.
+ * @returns {any} The values, potentially locale substituted.
+ */
+
+export function _substituteLocalizations(values: any, localizations: any) : object {
```

Comment:
How about copying [the localizations type from nimbus shared](https://github.com/mozilla/nimbus-shared/blob/main/types/experiments.ts#L121-L131)?

I'm not quite sure what to do about the values typing. If there's something we could copy-paste, that'd be great.  If not, c'est la vie. Up to you...